src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Mac/ButtonExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AndroidX.AppCompat.Widget;
+
+namespace Xamarin.Platform
+{
+	public static class ButtonExtensions
+	{
+		public static void UpdateColor(this IButton button, NSButton button)
```

Comment:
Notice ButtonExtensions.cs doesn't have ".mac."  

It's sort of nice having ".mac." for navigation purposes but it seems a bit excessive to have /mac/file.mac.cs